front/app/modules/commercial/content_builder/admin/components/ContentBuilderPreview/Preview/index.tsx

```diff
@@ -56,6 +59,11 @@ const Preview = ({ projectId, projectTitle }: PreviewProps) => {
           <Editor isPreview={true}>
             <ContentBuilderFrame editorData={editorData} />
           </Editor>
+          {!isNilOrError(projectFiles) && (
+            <Box maxWidth="750px">
```

Comment:
One minor one, do we need the max-width here?

Reply:
Yeah, Ben didn't want the attachments to be the full width of the content builder description. So I've used a max-width which is similar to how wide the attachments are in the regular, non-content-builder description. Is it bad practice to set it like this?

Reply:
No, I feel 100% would have been better but let's go with this since Ben recommended this